.storybook/stories/Input/Input.tsx

```diff
@@ -67,7 +118,6 @@ const useStyles = (
       color: theme.colors[textColor],
       borderColor: theme.colors[borderColor],
       width: '100%',
-      ...theme.typography['body-medium'],
```

Comment:
Is there any specific reason why you removed typography from useStyles hook and moved it to the style object inside a component? I think here it's a better place

Reply:
Because it's causing errors. Now everything seems fine and vscode doesn't show errors.